src/domain/position/binProcessor.js

```diff
@@ -146,12 +157,24 @@ const processBins = async (bins, trx) => {
     let accumulatedTransferStateChanges = []
     let accumulatedPositionChanges = []
 
-    // If non-prepare action found, log error
-    // We need to remove this once we implement all the actions
-    if (actions.length > 1 || (actions.length === 1 && actions[0] !== 'prepare')) {
-      Logger.isErrorEnabled && Logger.error('Only prepare action is allowed in a batch')
-      // throw new Error('Only prepare action is allowed in a batch')
-    }
```

Comment:
I think we still need to check for prepare and commit messages and throw an error if different action found in batch topic.

Reply:
Is it not enough that we ask the user to config the topic overrides?
Any non prepare/commit action gets binned by action so they would never be accidently passed into a function right?

Reply:
If we get other message in the batch topic, then we commit the offset and there will be no errors anywhere in the system. So I think we need this.